framework/h5py_interface_creator.py

```diff
@@ -140,8 +139,9 @@ def __isGroup(self,name,obj):
     """
     if isinstance(obj,h5.Group):
       self.allGroupPaths.append(name)
-      try   : self.allGroupEnds[name]  = obj.attrs["EndGroup"]
-      except:
+      try:
+        self.allGroupEnds[name]  = obj.attrs["EndGroup"]
+      except KeyError:
```

Comment:
Okay, thank for fixing this.  If you want to back out the other commit ( git checkout c54daf02a114ac7a9975d6093e764bf8bf987aa5; git checkout -b alfoa/addedMemoryHandlingForBechlerLikeCluster ) I can approve it.